### DISCUSSION TITLE
MIDAS - Retrieve spot metrics (e.g. FWHM) without running a full far field reconstruction

### DISCUSSION THREAD

**Discussion by jjsolano21** [marinerhemant/MIDAS#8] [Q&A]
Hi Hemant. I am trying to extract metrics such as FWHM from my HEDM far field data taken at 1ID, without having to run a full MIDAS far field reconstruction. Could you point me to what scripts I would need to call? Thanks! - Javi Solano, Purdue

**Comment by marinerhemant** [comment]
Hi Javi,

You can call the FF_HEDM/v7/ff_MIDAS.py code with **-peakSearchOnly 1** parameter to only run peaksearch.
